scripts/utils/formatCode.js

```diff
@@ -1,17 +1,16 @@
 "use strict";
-
+import prettier from "prettier";
 // TODO: Remove this `if` in Babel 8
 // Prettier only supports Node.js 10+, so we can fallback to not formatting
 // o CI on older Node.js versions
 
+let returnValue = {};
 if (process.env.CI && parseInt(process.versions.node, 10) < 10) {
```

Comment:
Since this file uses native ESM, it will only be run on Node.js >= 12. We can remove this `if`, keeping only the contents of the `else` branch.